Makefile

```diff
@@ -13,6 +13,15 @@ CWD ?= CURRENT_WORKING_DIRECTIONRY_NOT_SUPPLIED
 #  `VERBOSE_TEST="" make test_persistence` is an easy way to run the same tests without verbose output
 VERBOSE_TEST ?= -v
 
+# Detect OS using the $(shell uname -s) command
+ifeq ($(shell uname -s),Darwin)
```

Comment:
Should I install https://daoyuan.li/a-normal-sed-on-mac/?

Reply:
BSD sed is basically GNU sed with posix mode so no just makes format different